3-enrich/emr-etl-runner/config/config.yml.sample

```diff
@@ -45,6 +45,7 @@ aws:
       master_instance_type: m1.medium
       core_instance_count: 2
       core_instance_type: m1.medium
+      core_instance_bid: 0.015
```

Comment:
Should we add same comment as in `stream_config.yml.sample` here?

Reply:
I'm curious (forgive me if I'm wrong for making a comment here), but is there a reason you are requiring spot instance bids? I would expect this to be annoying to maintain. Does EMR not support a 'find lowest bid' approach, similar to several other AWS services (e.g., autoscaling groups)? Would it not be feasible to implement one? Cheers!

Reply:
Hi @ian-axelrod , thank you for making a comment. As you can see [here](https://github.com/snowplow/snowplow/pull/4123/files/4554517a3d5da24470335b3286e8f120a1bd71ea#diff-123bd97fb90bffdf6bcd93c5eb634da7), this change only gives the possibility to use spot instance bids, it doesn't force it. Some users of Snowplow pipeline have a need for it.